nautobot/extras/forms.py

```diff
@@ -791,6 +849,13 @@ class JobResultFilterForm(BootstrapMixin, forms.Form):
     )
 
 
+class ScheduledJobFilterForm(BootstrapMixin, forms.Form):
+    model = ScheduledJob
+    q = forms.CharField(required=False, label="Search")
+    name = forms.CharField(required=False)
+    total_run_count = forms.IntegerField(required=False)
```

Comment:
How will filtering on `total_run_count` work? Letting the user look for jobs that have run, say, exactly 1 time?

Reply:
Exactly. As of now it’s an exact lookup. I thought it might be most useful for things that ran less than a handful of times, but I don’t actually know.